lib/src/data/mapper/i_mappers.dart

```diff
@@ -0,0 +1,5 @@
+abstract class IMapper<FROM, TO> {
```

Comment:
Why do you need this mapper?

Reply:
In this particular case is the same to have a mapper or not, but i think there are cases in which the model and the entity are structured different, so a mapper is needed.